src/content/admin/dashboard.mdx

```diff
@@ -118,9 +118,9 @@ This table provides a consolidated view about preview environments, including:
 - Status (`Active` or `Sleeping`)
 - Last seen
 
-## Variables
+## Admin Variables
```

Comment:
This changes the anchors reference, could you do a replace of `dashboard.mdx#variables` -> `dashboard.mdx#admin-variables`